frontend/openchat-client/src/openchat.ts

```diff
@@ -4995,6 +4997,31 @@ export class OpenChat extends OpenChatAgentWorker {
         });
     }
 
+    submitProposal(governanceCanisterId: string, proposal: CandidateProposal): Promise<boolean> {
+        return this.sendRequest({
+            kind: "submitProposal",
+            governanceCanisterId: Principal.fromText(governanceCanisterId),
+            proposal,
+        }).then((resp) => {
+            if (resp.kind === "success" || resp.kind === "retrying") {
```

Comment:
Seems a bit fishy that `retrying` and `success` mean the same thing. Doesn't really _seem_ like they are the same.

Reply:
There isn't really much I can do - the backend will keep retrying indefinitely and should succeed. Seeing the new proposal is anyway asynchronous.